app/services/submit_application_with_decoupled_references.rb

```diff
@@ -0,0 +1,28 @@
+class SubmitApplicationWithDecoupledReferences
+  attr_reader :application_form, :application_choices
+
+  def initialize(application_form)
+    @application_form = application_form
+    @application_choices = application_form.application_choices
+  end
+
+  def call
+    # TODO: the decoupled_references feature makes the edit window redundant.
+    # Drop the ApplicationForm#edit_by column when removing the
+    # decoupled_references feature flag.
+    application_form.update!(
+      submitted_at: Time.zone.now,
+      edit_by: Time.zone.now,
```

Comment:
This makes sense for now, but I'm wondering if it should be set to nil when the flag is removed? Not that it's a big deal either way. We could potentially just drop the column though since it won't be used for anything going forward.

Reply:
Worth discussing when removing the flag :+1: